packs/github/actions/add_team_membership.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: add_team_membership
```

Comment:
add_team_member is an existing action, which is being deprecated in the GitHub API. Depends on whether the user knows the github API!